### PR TITLE
Combine Label and Label Group docs

### PR DESCRIPTION
**Pull request by mcarrano**
Combines the docs for Label and Label group. I renamed and added some additional text to explain and better differentiate some of the examples.

**What**: Closes https://github.com/patternfly/patternfly-react/issues/8543

**Additional issues**:

### CODE REVIEW

**Review by nicolethoen — APPROVED** [review]
LGTM

**Review by tlabaj — APPROVED** [review]
LGTM

File: packages/react-core/src/components/Label/examples/Label.md [hunk 1]
```diff
@@ -38,3 +38,45 @@ You can also customize any Label's close button aria-label as this example shows
...
+```
+
+### Editable label group
+
+```ts isBeta file="LabelGroupEditableLabels.tsx"
+```
+
+### Editable label group with add button
+
+The contents of a label group can be modified by removing labels or adding new ones using the Add button. For additional documentation that showcases adding a new label, see [label group demos](/components/label/react-demos).
```
Comment (edonehoo) [inline packages/react-core/src/components/Label/examples/Label.md:79]:
```suggestion
The contents of a label group can be modified by removing labels or adding new ones using the Add button. For additional documentation that showcases adding a new label, see [label demos](/components/label/react-demos).
```

**Review by edonehoo — COMMENTED** [review]
I only have one change suggestion since the phrasing "label group demos" doesn't align with combining label + label group into one component. 

I would also recommend adding "labels" to the end of the standard label examples to be in line with other documentation we've updated. For example "Filled labels", "Outlined labels", etc.  However, my plan was to prioritize Labels and Chips as my next React doc PRs (within the next couple of weeks) to give them a full content review/update, so I could also just make those heading updates then.

**Comment by mcarrano** [comment]
@edonehoo I made updates based on your review. Please give this another look.

**Review by edonehoo — APPROVED** [review]

**Pull request merged**